app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -0,0 +1,47 @@
+import { componentTypes, validatorTypes, parseCondition } from '@@ddf';
+
+const createSchema = (fields, promise, edit, loadSchema) => {
+    const idx = fields.findIndex((field) => field.name === 'volume_type');
+
+    return ({
+        fields: [
+            {
+            component: componentTypes.SELECT,
+            name: 'ext_management_system_id',
+            id: 'ext_management_system_id',
+            label: __('Storage Manager'),
+            placeholder: __('<Choose>'),
+            onChange: value => API.options(`/api/cloud_volumes?ems_id=${value}`).then(loadSchema()),
+            loadOptions: () => promise.then(({ resources }) => resources.map(({ id, name }) => ({ value: id, label: __(name) }))),
+            isDisabled: edit,
+            isRequired: true,
+            validate: [{ type: validatorTypes.REQUIRED }],
+            },
+            {
+            component: componentTypes.TEXT_FIELD,
+            name: 'name',
+            id: 'name',
+            label: __('Volume Name'),
+            isRequired: true,
+            validate: [{ type: validatorTypes.REQUIRED }],
+            },
+            {
+                component: componentTypes.TEXT_FIELD,
+                name: 'edit',
+                id: 'edit',
+                type: 'hidden',
+                value: edit,
+            },
+            ...fields.slice(0, idx-1),
+            {
+                ...fields[idx],
+                resolveProps: ({ options }, _input, { getState }) => ({
+                    options: options.filter(({ condition }) => !condition || parseCondition(condition, getState().values)),
```

Comment:
Apparently `parseCondition` returns an object of two and you need just one of them:
```suggestion
                    options: options.filter(({ condition }) => !condition || parseCondition(condition, getState().values).result),
```